app/assets/stylesheets/grader.scss

```diff
@@ -107,47 +107,66 @@
 }
 
 
-/* Panes content (for controlling the heights) */
+/* Panes */
 
-#mark_viewer,
-#summary_viewer,
-#submission_rule_viewer,
-#codeviewer,
-#annotations_summary {
-  @include border-radius-bottom(5px);
-  overflow: auto;
-}
+#marking-pane {
+  margin-top: 1em;
 
-#codeviewer,
-#mark_viewer,
-#summary_viewer,
-#submission_rule_viewer,
-#annotations_summary,
-#remark_request_tab {
-  height: 650px;
-}
+  > div {
+    height: 800px;
 
+    @include breakpoint(mobile) {
+      display: block !important;
+      height: 100% !important;
+      padding: 0 !important;
+      width: 100% !important;
+    }
+  }
 
-/* Source code pane */
+  > div:first-child {
+    padding-right: 0.5em;
 
-// Adjusted height for users that also see the annotation menu
-#annotation_menu + #codeviewer {
-  height: 600px;
+    @include breakpoint(mobile) {
+      margin-bottom: 1em;
+    }
+  }
+
+  > div:last-child {
+    padding-left: 0.5em;
+  }
+
+  .jquery-dualpane-dragbar {
+    border: 1px solid $light-grey;
+    border-radius: 3px;
+
+    @include breakpoint(mobile) {
+      display: none !important;
+    }
+  }
 }
 
-.image_container {
+.tab-pane-contents > div {
+  @include border-radius-bottom(5px);
   overflow: auto;
-  width: auto;
+  height: 100%;
+}
+
+
+/* Source code pane */
+
+#image_container {
+  position: relative;
 }
 
 #sel_box {
   border: 1px dashed #f00;
+  cursor: crosshair;
   display: none;
   position: absolute;
 }
 
 .annotation_holder {
-  background: orange;
+  background: #ffa500;
```

Comment:
Color literals like `#ffa500` should only be used in variable declarations; they should be referred to via variable everywhere else.